proposals/scripts/template.md

```diff
@@ -28,7 +28,8 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 1. Copy this template to `new.md`, and create a commit.
 2. Create a GitHub pull request, to get a pull request number.
-    - Add the `proposal` and `WIP` labels to the pull request.
+    - Add the `proposal` label to the pull request and mark it as draft
+    - or ask someone to do it for you in case you can't.
```

Comment:
The intent is that people who are contributing should ask for access to the contributors group (which grants label permissions), i.e. following: https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md#collaboration-systems. I'd like to avoid "ask someone else to do it" instructions because it's not clear who, and it can also become a bit of churn as people become repeat contributors.

This should probably be more something like, at the top, "If you haven't already gotten contributor access, please [request it](https://github.com/carbon-language/carbon-lang/blob/trunk/CONTRIBUTING.md#collaboration-systems) in order to edit labels."